hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/services/AlertsService.java

```diff
@@ -218,6 +218,15 @@ void resolveAlertsForTrigger(String tenantId, String triggerId, String resolvedB
     void sendData(Collection<Data> data) throws Exception;
 
     /**
+     * Send data into the alerting system for evaluation.
+     *
+     * @param data Not Null.  The data to be evaluated by the alerting engine.
+     * @param ignoreFiltering  An optimization. Set true *only* if you are sure the data is useful for evaluation.
+     * @throws Exception any problem.
+     */
+    void sendData(Collection<Data> data, boolean ignoreFiltering) throws Exception;
```

Comment:
A configuration option would mean that every call from that client would be subjected to the same setting.  If you are blocking the PR on this I'll remove the option.  Although if the client can guarantee valid data I don't really see an issue. Metrics obviously is aware of filtering.  Let me know if this a blocking issue.